pygeoapi/util.py

```diff
@@ -341,7 +346,9 @@ def render_j2_template(config, template, data, locale_=None):
     env.filters['filter_dict_by_key_value'] = filter_dict_by_key_value
     env.globals.update(filter_dict_by_key_value=filter_dict_by_key_value)
 
-    # TODO: insert Babel Translation stuff here
+    translations = Translations.load('locale', [locale_])
+    env.install_gettext_translations(translations)
```

Comment:
We might need to check how efficient this is (loading all translations each time when `render_j2_template` is called). If the locale has not changed, there's no need to reload. But perhaps `install_gettext_translations` already has such a mechanism in place.

Reply:
> Good start @pvgenuchten and happy to see that it was easy to put the translations in place!
> For the links sections (as generated by `api.py`) we should have a look how to properly build in the Babel/gettext stuff there, but it might be better to do this after the API refactor, don't you agree @tomkralidis?

+1